apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingFactory.kt

```diff
@@ -213,14 +208,31 @@ class BehandlingFactory(
                             else -> null
                         },
                 )
-            behandlingHendelser.sendMeldingForHendelseMedDetaljertBehandling(
-                behandling.toStatistikkBehandling(persongalleri),
-                BehandlingHendelseType.OPPRETTET,
-            )
-            return BehandlingOgOppgave(behandling, oppgave)
+            return BehandlingOgOppgave(behandling, oppgave) {
+                behandlingHendelser.sendMeldingForHendelseMedDetaljertBehandling(
+                    behandling.toStatistikkBehandling(persongalleri),
+                    BehandlingHendelseType.OPPRETTET,
+                )
+            }
         }
     }
 
+    internal fun hentDataForOpprettBehandling(sakId: Long): OpprettBehandlingRequest {
```

Comment:
Muligens bare meg, men synes det kanskje blir litt misvisende navn, siden request-objektet vanligvis brukes for det som sendes inn i appen som dto. I `opprettSakOgBehandlingForOppgave` ender man opp med å bruke både `NyBehandlingRequest` og `OpprettBehandlingRequest`. Blir kanskje litt forvirrende? Kunne man hatt et navn som tilsier at dette er kontekst som er hentet, men som ikke er en del av selve requesten?